go.mod

```
@@ -28,7 +28,7 @@ require (
 	go.opencensus.io v0.24.0
 	go.uber.org/automaxprocs v1.5.3
 	go.uber.org/zap v1.24.0
-	golang.org/x/net v0.17.0
+	golang.org/x/net v0.23.0 //CVE-2023-45288
```

I like to include the version in the comment for reference
```suggestion
	golang.org/x/net v0.23.0 // CVE-2023-45288; >=v0.23.0
```